net-misc/gns3-gui/gns3-gui-2.2.8.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 desktop xdg-utils
+
+DESCRIPTION="Graphical Network Simulator"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="server"
```

Comment:
This doesn't do anything. 
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001